exist-core/src/main/java/org/exist/jetty/JettyStart.java

```diff
@@ -177,6 +177,9 @@ public synchronized void run(final String[] args, final Observer observer) {
                 System.getProperty("java.home", "(unknown java.home)")
             );
 
+            logger.info("Maximum amount of memory for JVM: {} MiB", Runtime.getRuntime().maxMemory() / (1024 * 1024));
```

Comment:
Whilst `MiB` might be more accurate. Could we just use `MB` so as to cause less confusion?

Also instead of doing the calculation yourself, you might like to use `FileUtils#humanSize(long)` as that will take care of MB, GB, etc.

Reply:
LGTM otherwise :-)